### DISCUSSION TITLE
Anyone uses Goldendict? Please help

### DISCUSSION THREAD

**Discussion by Aribeth07** [end-4/dots-hyprland#775] [Q&A]
Installed [goldendict-ng ](https://aur.archlinux.org/packages/goldendict-ng). The "translate word from clipboard" function (hotkey ctrl+c+c) isn't working when goldendict loses focus. 

I mean if I open goldendict, pick a word INSIDE the UI and hit ctrl+c+c, it works. But if I pick a word from a document that is opened in the same workspace or other workspace, it does nothing (unless I move the cursor back to goldendict and press the hotkey again)
Any tweak on the config files I can do to get a work around?

**Comment by end-4** [comment]
clipboard not working properly sounds like an xwayland problem
